docs/section19/19-26.md

```diff
@@ -0,0 +1,99 @@
+        - Check if each value in the design_heating_multiplier_schedule aligns with the int_lgt_min_value: `for x in list(int_lgt_design_heating_multiplier_sch):`
+            - Check if the hourly value does not equal the lowest hourly value used in the multiplier schedule: `if x != int_lgt_min_value: int_lgt_pass_heating = false`  
+        - Check if each value in the design_cooling_multiplier_schedule aligns with the int_lgt_max_value: `for x in list(int_lgt_design_cooling_multiplier_sch):`
+            - Check if the hourly value does not equal the highest hourly value used in the multiplier schedule: `if x != int_lgt_max_value: int_lgt_pass_cooling = false`  
+
+    Conduct checks for the miscellaneous objects:  
+    - Get list of misc equipment objects: `misc_obj_list = list(space.miscellaneous_equipment)`   
+    - Reset misc_pass_heating boolean variable: `misc_pass_heating = true`   
+    - Reset misc_pass_cooling boolean variable: `misc_pass_cooling = true`   
+    - For each misc equipment object: `for misc in misc_obj_list:`  
+        - Get the multiplier schedule: `misc_multiplier_sch = get_component_by_id(B_RMI,misc.multiplier_schedule)`  
+        - Get the design_heating_multiplier_schedule: `misc_design_heating_multiplier_sch = misc_multiplier_sch.heating_design_day_sequence`  
+        - Get the design_cooling_multiplier_schedule: `misc_design_cooling_multiplier_sch = misc_multiplier_sch.cooling_design_day_sequence`  
+        - Get the highest hourly value used in the multiplier schedule (not sure if the schedule needs to be transformed or anything, for example, what if it is an event schedule?): `misc_max_value = max(misc_multiplier_sch.hourly_values)`   
+        - Get the lowest hourly value used in the multiplier schedule (not sure if the schedule needs to be transformed or anything, for example, what if it is an event schedule?): `misc_min_value = min(misc_multiplier_sch.hourly_values)` 
+        - Check if each value in the design_heating_multiplier_schedule aligns with the misc_min_value: `for x in list(misc_design_heating_multiplier_sch):`
+            - Check if the hourly value does not equal the lowest hourly value used in the multiplier schedule: `if x != misc_min_value: misc_pass_heating = false`  
+        - Check if each value in the design_cooling_multiplier_schedule aligns with the misc_max_value: `for x in list(misc_design_cooling_multiplier_sch):`
+            - Check if the hourly value does not equal the highest hourly value used in the multiplier schedule: `if x != misc_max_value: misc_pass_cooling = false`  
+
+    - **Rule Assertion:** 
+    - Case 1: If the space type is not dwelling unit and all schedules pass per above then pass: `if is_dwelling_unit == false and all(inf_pass_heating, inf_pass_cooling,occ_pass_heating, occ_pass_cooling,int_lgt_pass_heating, int_lgt_pass_cooling,misc_pass_heating, misc_pass_cooling) == true: outcome = "PASS"`  
```

Comment:
I think if building type is multifamily AND space types are not specified, we should provide UNDETERMINED outcome and a message descibing design day schedules.

Reply:
CML to update